config-cross-recipe.sh

```diff
@@ -1,156 +0,0 @@
-#!/bin/bash
```

Comment:
now ACC scripts are modified to pick config-cross-recipe.sh from scripts/es2k/config-cross-recipe.sh
Older reference can be removed

Reply:
Cool! Can you approve the PR, so I can merge it?
I'll remove the symlinks in a follow-up commit.